go-controller/pkg/ovn/gateway_init.go

```diff
@@ -142,9 +166,6 @@ func (oc *Controller) gatewayInit(nodeName string, clusterIPSubnet []*net.IPNet,
 		return fmt.Errorf("failed to add logical router port %q for gateway router %s, err: %v", gwRouterPort, gatewayRouter, err)
 	}
 
-	ctx, cancel := context.WithTimeout(context.Background(), types.OVSDBTimeout)
```

Comment:
is this not used anymore?

Reply:
no we moved it upto L75 where I use it to fetch the router.